src/MissionParallelCatchup/parallel_catchup_helm/files/logarithmic_range_generator.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
If you've got python on these hosts, I would consider writing some or all of these scripts in python. Bourne shell works of course, but even simple things quickly get hard and brittle and difficult to debug.

Reply:
unfortunately the range generator is based off the Redis image and it doesn't contain Python.

Reply:
as we try other more complex range strategies, it'd be worth it to move all these different strategies inside a separately Python container.